ee/docker-compose.ch.yml

```diff
@@ -58,6 +58,7 @@ services:
             PGHOST: db
             PGUSER: posthog
             PGPASSWORD: posthog
+            CAPTURE_INTERNAL_METRICS: 'true'
```

Comment:
Q: Why?

Reply:
that was just for testing, planning to remove these

Reply:
just testing, will remove